pkg/job/job.go

```diff
@@ -175,3 +192,22 @@ func (j Job) JUnit() (io.Reader, error) {
 	}
 	return j.fetch(u)
 }
+
+func (j Job) ToHtml() string {
+	startTime, _ := j.StartTime()
```

Comment:
ah, now I see why you're panicking in StartTime :joy: 

May I suggest that you either:
* handle service degradation (thus ignoring the error, but without panicking); OR
* guarantee the result and bubble up the error?

Reply:
worst case, you can panick _here_